scripts/elevators.zs

```diff
@@ -0,0 +1,74 @@
+import crafttweaker.liquid.ILiquidStack;
+import crafttweaker.item.IItemStack;
+
+// remove
+recipes.removeByMod("elevatorid");
```

Comment:
Don't use this. Remove them individually instead, preferably by recipe name and loops.